src/Apps/Order/redirects.tsx

```diff
@@ -161,25 +184,34 @@ export const shouldRedirect = ({
     return false
   }
 
-  const locationParts = location.pathname.split("/").slice(3)
-
-  function traverse(node: RedirectRecord, path: string[]): void {
+  function traverse(node: RedirectRecord, path: string): void {
```

Comment:
Do you think the current test coverage in `redirects` is enough to handle the new logic that we're adding here?

It seems to cover our current cases but I'm wondering if we should add some unit-level tests for this method.

Reply:
I think that the test coverage would be good enough if we add a test for the "review/counter" route, but also that unit tests which explicitly talk about this method would be really helpful for people who need to modify it in the future.

Reply:
How do you feel about us doing that in a separate PR?

Reply:
Seems 👍 !

Reply:
Added a ticket: https://artsyproduct.atlassian.net/browse/PURCHASE-717